server/src/TPlayer/TPlayer.cpp

```diff
@@ -2026,21 +2026,16 @@ bool TPlayer::addWeapon(LevelItemType defaultWeapon)
 	if ( !settings->getBool("defaultweapons", true))
 		return false;
 
+    // TODO: (samich) - is this creating a default weapon for the server if the one of the
```

Comment:
for older clients (v1/v2) default weapons scripts were on the client so we just needed to acknowledge they exist. iirc, change looks good.

good catch